addons/website_event/static/src/interactions/modal_registration.js

```diff
@@ -4,8 +4,8 @@ import { registry } from "@web/core/registry";
 import { session } from "@web/session";
 import { Interaction } from "@web/public/interaction";
 
-export class ModalAttendeesRegistration extends Interaction {
-    static selector = "#modal_attendees_registration";
+export class ModalRegistration extends Interaction {
+    static selector = "#modal_attendees_registration,#modal_slot_ticket_registration";
```

Comment:
Problem with dynamic IDs building is that it doesn't give any result when greping, making you think it's unused (related to previous comment).